adapters/defaultLogger/defaultLogger.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2016 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package defaultLogger
+
+import (
+	"errors"
+	"os"
+	"text/template"
+
+	"istio.io/mixer"
+	"istio.io/mixer/adapters"
+)
+
+const (
+	name = "istio.io/mixer/loggers/defaultLogger"
+	desc = "Creates access log entries from Report() requests and flushes to os.Stdout"
+	tmpl = "defaultLogger"
+)
+
+type (
+	builder struct{}
+	adapter struct {
+		t *template.Template
+	}
+
+	// Config represents the adapter configuration for the default logger.
+	Config struct {
+		// Template is used to pass a golang text/template string to the
+		// adapter for translation of fact values into a log entry.
+		Template string
```

Comment:
They are very easy to follow and built into the language. 
Only preculiarity is  {{ .attribute }} accessor. However, since all our variables are top level, it will look natural.